virtool/groups/data.py

```diff
@@ -78,10 +78,22 @@ async def update(self, group_id: str, permissions_update: Dict[str, str]) -> Gro
         if not permissions:
             raise ResourceNotFoundError
 
+        permissions_update = data["permissions"]
+
+        name = group_id
+
+        if "name" in data:
+            name = data["name"]
```

Comment:
If name is not in data, just do nothing. The response model already populates `name` with `id` if no name is provided.

Please ask me if this is not clear. It is important.